File: Cpp/Operators/other.cpp
#include <iostream>

using namespace std;

int main() {
    // comma operator
    int a, b = 20, c;

    a = 10;
    c = a + b;
    cout << "c = " << c << endl;

    // sizeof operator
    cout << "Size of c: " << sizeof(c) << " bytes" << endl;

    // address of operator
    cout << "Address of c: " << &c << endl;

    // tenary operator "? :" is a ternary operator which works as follows: condition ? expression1 : expression2;
    int x = 10, y = 20;
    int z = (x > y) ? x : y;
    cout << "Value of z: " << z << endl;

    // << and >> operators
    // cout << x; // output
    // cin >> x; // input

    // Member access operator
    // . and -> are member access operators

    return 0;
}

File: Python/Operators/other.py
def main():
    # identity operators
    x = 5
    y = 5
    x1 = "Hello"
    y1 = "Hello"
    x2 = [1, 2, 3]
    y2 = [1, 2, 3]
    print(x is y)
    print(x is not y)
    print(x1 is y1)
    print(x1 is not y1)
    print(x2 is y2)
    print(x2 is not y2)
    print()
    print(id(x))
    print(id(y))
    print(id(x1))
    print(id(y1))
    print(id(x2))
    print(id(y2))
    print()

    # membership operators
    x = ["apple", "banana"]
    print("banana" in x)
    print("pineapple" not in x)

    # address operator
    print(id(x))
    print()

    # type operator
    print(type(x))
    print()

    # size operator
    print(len(x))

if __name__ == "__main__":
    main()